frontend/public/components/markdown-view.tsx

```diff
@@ -67,6 +72,10 @@ export class SyncMarkdownView extends React.Component<
 
     // Let the new height take effect, then reset again once we recompute
     setTimeout(() => {
+      if (this.unmounted) {
+        return;
+      }
```

Comment:
Prefer to clear the timeout instead.
```suggestion
    this.timeoutHandle = setTimeout(() => {
```

in `componentWillUnmount` call `clearTimeout(this.timeoutHandle)`

Reply:
Done.